test/e2e/nftables_test.go

```diff
@@ -45,20 +70,36 @@ var _ = Describe("Nftables", func() {
 			masterNFT, err = AddPortsToNFTables(masterNFT, extraNFTablesFile)
 			Expect(err).NotTo(HaveOccurred())
 		}
-
 		g := new(errgroup.Group)
-		for _, node := range nodeList.Items {
```

Comment:
Why not keeping this loop to avoid duplicating the MC? It's checking the node role first

Reply:
but my MC is per role not per node